lucien-ios/Models/Dashboard.swift

```diff
@@ -9,19 +9,18 @@
 import Foundation
 
 struct Dashboard: Decodable {
+
     var myCollectionCount: Int
+    var lendingCount: Int
+    var borrowingCount: Int
+    var lending: [Comic]
```

Comment:
This naming is not very descriptive. What is being lent? What is being borrowed?